test/src/e2e_vm_tests/test_programs/should_pass/language/intrinsics/dbg/stdout.snap

```diff
@@ -2,12 +2,112 @@
 source: test/src/snapshot/mod.rs
 ---
 > forc build --path test/src/e2e_vm_tests/test_programs/should_pass/language/intrinsics/dbg --asm final | grep ecal
-ecal $r2 $r3 $r0 $r1          ; ecal id fd buf count
+ecal $r4 $r2 $r3 $r0          ; ecal id fd buf count
+ecal $r2 $r1 $zero $zero      ; ecal id fd zero zero
+ecal $r3 $r0 $r1 $r2          ; ecal id fd buf count
 ecal $r3 $r0 $r1 $r2          ; ecal id fd buf count
 ecal $r3 $r0 $r1 $r2          ; ecal id fd buf count
 ecal $r3 $r0 $r1 $r2          ; ecal id fd buf count
-ecal $r4 $r2 $r3 $r0          ; ecal id fd buf count
-ecal $r2 $r1 $zero $zero      ; ecal id fd zero zero
 ecal $r3 $r0 $r1 $r2          ; ecal id fd buf count
+ecal $r2 $r0 $r1 $r3          ; ecal id fd buf count
+ecal $r3 $r1 $r2 $r4          ; ecal id fd buf count
 
 > forc build --path test/src/e2e_vm_tests/test_programs/should_pass/language/intrinsics/dbg --release --asm final | grep ecal
+
+> forc test --path test/src/e2e_vm_tests/test_programs/should_pass/language/intrinsics/dbg --logs
+exit status: 0
+output:
+    Building test/src/e2e_vm_tests/test_programs/should_pass/language/intrinsics/dbg
+   Compiling library std (sway-lib-std)
+   Compiling script dbg (test/src/e2e_vm_tests/test_programs/should_pass/language/intrinsics/dbg)
+    Finished debug [unoptimized + fuel] target(s) [37.368 KB] in ???
+     Running 1 test, filtered 0 tests
+
+tested -- dbg
+
+      test call_main ... ok (???, 103315 gas)
```

Comment:
```suggestion
      test call_main ... ok (???, 103315 gas)
```

Is this harcoded (103315) gas amount going to create issues if something changes about the gas usage for the specific opcodes etc in a later fuel-vm, or is this test isolated enough that a change in the gas amount not going to be an issue either from an optimization from compiler side, or a change from vm side?

Reply:
I think the idea is to force the snapshot to break, if gas usage change, so we are aware of what is happening.

If this becomes meaningless at somepoint we may remove it from the snapahot later.